### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by viktorprogger**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ✔️
| Fixed issues  | ❌

### CODE REVIEW

**Review by samdark — APPROVED** [review]

**Review by vjik — COMMENTED** [review]
Need raise PHP version to 8

File: src/Client/Facebook.php [hunk 1]
```diff
@@ -38,17 +38,17 @@
  */
 final class Facebook extends OAuth2
 {
-    private string $authUrl = 'https://www.facebook.com/dialog/oauth';
-    private string $tokenUrl = 'https://graph.facebook.com/oauth/access_token';
-    private string $endpoint = 'https://graph.facebook.com';
+    protected string $authUrl = 'https://www.facebook.com/dialog/oauth';
```
Comment (vjik) [inline src/Client/Facebook.php:41]: Why "protected"? In final classes "private" equal to "protected".

Comment (samdark) [reply src/Client/Facebook.php:41]: Inheritance?

Comment (vjik) [reply src/Client/Facebook.php:41]: ha-ha, exactly 😀

**Pull request merged**